### PR TITLE
fix layout classes

### PR DESCRIPTION
**Pull request by notwaldorf**
Fixes https://github.com/PolymerElements/paper-input/issues/10

### CODE REVIEW

**Comment by notwaldorf** [comment]
@morethanreal yesterday's changes to layout broke `paper-input`, and this fixes it. PTAL.

File: bower.json [hunk 1]
```diff
@@ -23,7 +23,9 @@
   "dependencies": {
     "polymer": "Polymer/polymer#v0.8.0-rc.7",
     "iron-input": "PolymerElements/iron-input#^0.8.0",
-    "paper-styles": "PolymerElements/paper-styles#^0.8.0"
+    "iron-autogrow-textarea": "PolymerElements/iron-autogrow-textarea#^0.8.0",
```
Comment (morethanreal) [inline bower.json]: Where is this used?

Comment (notwaldorf) [inline bower.json]: Argh, that's my checkout being wonky. Doesn't need to be here yet.

**Comment by morethanreal** [comment]
Since `paper-styles` has a dependency on `iron-flex-layout`, shouldn't it get pulled in automatically?

**Comment by notwaldorf** [comment]
Like this? The `relative` class gets added imperatively, so I thought it was easier to just make the class.

**Comment by morethanreal** [comment]
I was thinking add these rules:

```css
.input-content {
    position: relative;
}
.underline {
    position: relative;
}
```

And in `_computeUnderlineClass` return `underline` instead of relative, and remove `relative` from value returned in `_computeInputContentClass`.

**Comment by notwaldorf** [comment]
Done! :computer:

**Comment by morethanreal** [comment]
lgtm

**Pull request merged**